allennlp/common/from_params.py

```diff
@@ -564,6 +564,17 @@ def from_params(
         else:
             # This is not a base class, so convert our params and extras into a dict of kwargs.
 
+            # If there was a "type" given, we probably didn't want to end up here.
+            if "type" in params:
+                value = params.pop("type")
+                message = (
+                    f"{value} not in acceptable choices for {params.history}type. "
```

Comment:
Missing space before "type"?

Reply:
No, `params.history` already ends in a period. I know, it looks weird ...